components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -0,0 +1,123 @@
++++
+title = "Data Feed"
+description = "Exporting Node Data from Chef Client Runs to External Data Collectors"
+date = 
+draft = true
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "settings"
+    weight = 10
++++
+
+## About Automate Data Feed
+
+The Data Feed service is used to send node data to a 3rd party service. This can be useful when updating Configuration Management Databases, external Security Dashboards and ITSM platforms.
+The following types of information are sent:
+ 
+ - Ohai data gathered from each node being managed, this includes HW, OS and installed program information, there is some variation depending on the OS being managed
+ - Configuration information about each node being managed, this includes Chef Client Run status, Runlists, Cookbooks, Recipes being run against each node
+ - Compliance information about each node to show it’s compliance state, this includes passed and failed controls for each profile executed against the node
+
+The Data Feed is not a Data Tap, it operates by doing the following:
+
+- Every 30 minutes (configurable) the data-feed-service will aggregate the client runs and compliance reports from the previous 30 minutes and send this to the registered destinations.
+- If there are no destinations, aggregation will not occur.
+- The data is aggregated and sent in batches of 100 nodes at a time (also configurable)
+
+
+By default only Admins of Chef Automate may create and manage Data Feeds.
+
+### Adding a Data Feed Endpoint
+
+To add an endpoint for a Data Feed in Chef Automate:
+
+![Settings Page](/images/docs/settings_data_feed.png)
+
+![Setup Data Feed Page](/images/docs/filled_form_create_data_feed.png)
+
+1. In the **Settings** tab, navigate to the _Data Feed_ page in the sidebar.
+1. Select **Create Data Feed**.
+1. Enter a unique Data Feed name.
+1. Enter the URL for your Data Feed End Point, including any specific port details
+1. Enter the Username and Password that your 3rd party endpoint requires for Authentication
+1. Use the **Test Data Feed** button to validate the connection details
+1. Once the Test is successful, use the **Create Data Feed** button to save your Data Feed Configuration
+
+![Saved Data Feed Page](/images/docs/saved_data_feed.png)
```

Comment:
weve been trying to limit the number of screenshots to only ones that are totally necessary, i think you could drop the saved, edit, and delete screenshots.

Reply:
Thanks @susanev - pulled them back out